INSTALL.md

```diff
@@ -491,15 +491,18 @@ finite) which are required for the available algorithms. See section
 "ApplicableMethod" and "KnownPropertiesOfObject" of the GAP Reference
 manual.
 
-Problems specific to Windows
-
 ### The ^-key or "-key cannot be entered.
 
 This is a problem if you are running a keyboard driver for some non-english
-languages. These drivers catch the ^ character to produce the French
-circumflex accent and do not pass it properly to GAP. No fix is known. (One
+languages. These drivers catch the ^ character to produce the French circumflex
+accent and do not pass it properly to GAP. For OS X users, as a workaround
+please refer to the section "GAP for OS X" below for information on
+how to install readline and section 5 on how to recompile GAP, for windows no
+fix is known. (One
 can type POW(a,b) for a^b.)
 
+Problems specific to Windows
```

Comment:
```suggestion
## Problems specific to Windows
```
I quickly looked at how this renders, and I'd suggest to indeed changes this. After this, I am only wondering if the separation into three commits here really is warranted, perhaps you could briefly explain it, or alternatively, squash it down. But that's a minor concern, in principle I'd be fine with this to be merged :-). Would be good to hear if @alex-konovalov has any further input, though.